packages/chain-adapters/src/evm/gnosis/GnosisChainAdapter.ts

```diff
@@ -0,0 +1,118 @@
+import type { AssetId } from '@shapeshiftoss/caip'
+import { ASSET_REFERENCE, gnosisAssetId } from '@shapeshiftoss/caip'
+import type { BIP44Params } from '@shapeshiftoss/types'
+import { KnownChainIds } from '@shapeshiftoss/types'
+import * as unchained from '@shapeshiftoss/unchained-client'
+
+import type { FeeDataEstimate, GetFeeDataInput } from '../../types'
+import { ChainAdapterDisplayName } from '../../types'
+import { bn, bnOrZero } from '../../utils'
+import type { ChainAdapterArgs } from '../EvmBaseAdapter'
+import { EvmBaseAdapter } from '../EvmBaseAdapter'
+import type { GasFeeDataEstimate } from '../types'
+
+const SUPPORTED_CHAIN_IDS = [KnownChainIds.GnosisMainnet]
+const DEFAULT_CHAIN_ID = KnownChainIds.GnosisMainnet
+
+export class ChainAdapter extends EvmBaseAdapter<KnownChainIds.GnosisMainnet> {
```

Comment:
not the most useful tests, but should still probably add the chain adapters tests for gnosis chain adapter

Reply:
https://github.com/shapeshift/web/commit/4ca7e68a7232b3366fc974ec1d8445f5a7d0a744